packages/faustwp-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@faustwp/core",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
Is there a reason we are incrementing the version manually here? If not we should revert this and regenerate the `package-lock.json`